lunar/distribution.yaml

```diff
@@ -58,6 +58,26 @@ repositories:
       url: https://github.com/ros/angles.git
       version: master
     status: maintained
+  ar_track_alvar:
+    doc:
+      type: git
+      url: https://github.com/sniekum/ar_track_alvar.git
+      version: kinetic-devel
+    release:
+      packages:
+      - ar_track_alvar
+      - ar_track_alvar_metapkg
+      - ar_track_alvar_msgs
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/ros-gbp/ar_track_alvar-release.git
+      version: 0.7.0-0
+    source:
+      test_pull_requests: true
```

Comment:
@130s Actually I doubt that you have set up the necessary access to this personal repo. If that is not the case please remove this flag in a follow up PR.